frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.tsx

```diff
@@ -297,6 +300,8 @@ function PivotTable({
 
   const topHeaderHeight = topHeaderRows * CELL_HEIGHT;
 
+  const queryBuilderGridHeight = heightProp - topHeaderHeight;
```

Comment:
Wdyt about `pivotTableBodyHeight`?

Reply:
I prefer using `queryBuilder` in the name to signify that this variable is only meant to be used in the query builder (e.g. when `isDashboard` is false), otherwise we might forget and accidentally use it where it shouldn't be.